js/bybit.js

```diff
@@ -6428,9 +7165,7 @@ module.exports = class bybit extends Exchange {
     }
 
     async setMarginMode (marginMode, symbol = undefined, params = {}) {
-        if (symbol === undefined) {
-            throw new ArgumentsRequired (this.id + ' setMarginMode() requires a symbol argument');
-        }
+        this.checkRequiredSymbol ('setMarginMode', symbol);
```

Comment:
@sc0Vu shouldn't this condition be `&&` instead of `||` ?

Reply:
### 
![image](https://user-images.githubusercontent.com/43336371/220911752-2bc59c47-df20-4400-b9c0-4b4b69844623.png)